lib/grizzly/zwave/commands/switch_multilevel_report.ex

```diff
@@ -44,9 +44,8 @@ defmodule Grizzly.ZWave.Commands.SwitchMultilevelReport do
   end
 
   def encode_value(:off), do: 0x00
-  def encode_value(value) when value in 0..100, do: value
+  def encode_value(value) when value in 0..99, do: value
   def encode_value(:unknown), do: 0xFE
-  def encode_value(:previous), do: 0xFF
```

Comment:
I think this was accidentally deleted.

Reply:
Also, I think that it should be possible to specify 0xff and not raise.

Reply:
Actually, the deletion is intentional. A multilevel switch never reports the current level as :previous. (This was originally from copy-pasting from the set command). 0xFF (deprecated) means 100% hardware level, thus 99.

Reply:
Agreed on not raising on 0xFF

Reply:
0xFF means previous not 99